framework/physics/FieldFunction/ff_gridbased_00_constrdestr.cc

```diff
@@ -36,25 +50,12 @@ chi::InputParameters FieldFunctionGridBased::GetInputParameters()
 FieldFunctionGridBased::FieldFunctionGridBased(
   const chi::InputParameters& params)
   : FieldFunction(params),
+    sdm_(MakeSDM(params)),
+    ghosted_field_vector_(MakeFieldVector(*sdm_, UnkManager())),
```

Comment:
It would be good to see `UnknownManager` as the function name instead

Reply:
Will make it `GetUnknownManager` other UnknownManager conflicts with the type.